src/gettime.h

```diff
@@ -30,13 +30,18 @@ enum TimePrecision
 	PRECISION_NANO
 };
 
-inline std::string getTimestamp()
+inline std::string getTimestamp(std::string fmt)
```

Comment:
const char * please, we don't need c++ there it's a const raw C parameter

Reply:
I'll make the change and send it to MoNTE48.

Reply:
This is changed now.